plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -292,6 +307,7 @@ def _onAddDevice(self, name, address, properties):
         if global_container_stack and device.getId() == global_container_stack.getMetaDataEntry("um_network_key"):
             # Ensure that the configured connection type is set.
             global_container_stack.addConfiguredConnectionType(device.connectionType.value)
+            # global_container_stack.setFirmwareVersion(device.firmwareVersion)
```

Comment:
In connection to the comment above... we could set the firmware in the global stack so that the machine manager can access it, but why? Either way we still have spaghetti.

Reply:
The main reason we set the configuredConnection type is because we want that available if there is no connection. If we don't care about the firmware version if there is no connection, there is no point in storing it.

Reply:
Makes sense, except that there often is a connection and then the firmware is a bitch to get...